src/types/userType.ts

```diff
@@ -0,0 +1,6 @@
+export interface MyProfileResponse {
+  userId: 'number';
+  nickname: 'string';
+  profileImg: 'string';
+  OAuth: 'string';
+}
```

Comment:
```suggestion
export interface MyProfileResponse {
  userId: number;
  nickname: string;
  profileImg: string;
  OAuth: string;
}
```

![image](https://user-images.githubusercontent.com/76927397/211042965-9bb346fa-15ca-4e45-a166-a8d31c1fb92c.png)


<br>

`number` 또는 `string`이라는 타입이 아니라

`'number'` , `'string'` 이라는 문자열로 타입이 선언된 상태입니다! 

수정해주시면 될 것 같습니다!

Reply:
Oops

Reply:
@pds0309 감사합니다! 수정해서 커밋했습니당

Reply:
앞으로 제가 더 많은 실수를 할 테니 잘 부탁드립니다 👍 👍